_data/quality_estimation.json

```diff
@@ -4172,5 +4173,341 @@
+            "ht",
+            "hu",
+            "hy",
+            "id",
+            "ig",
+            "ilo",
+            "is",
+            "it",
+            "ja",
+            "jv",
+            "ka",
+            "kab",
+            "kac",
+            "kam",
+            "kas",
+            "kbp",
+            "kea",
+            "kg",
+            "khk",
+            "ki",
+            "kk",
+            "km",
+            "kmb",
+            "kmr",
+            "kn",
+            "knc",
+            "ko",
+            "ks",
+            "ky",
+            "la",
+            "lb",
+            "lg",
+            "li",
+            "lij",
+            "lmo",
+            "ln",
+            "lo",
+            "lt",
+            "ltg",
+            "lua",
+            "luo",
+            "lus",
+            "lv",
+            "lvs",
+            "mag",
+            "mai",
+            "mg",
+            "mi",
+            "min",
+            "mk",
+            "ml",
+            "mn",
+            "mni",
+            "mos",
+            "mr",
+            "ms",
+            "mt",
+            "my",
+            "nb",
+            "ne",
+            "nl",
+            "nn",
+            "nso",
+            "nus",
+            "ny",
+            "oc",
+            "or",
+            "pa",
+            "pag",
+            "pap",
+            "pbt",
+            "pes",
+            "pl",
+            "plt",
+            "prs",
+            "ps",
+            "pt-br",
+            "pt-pt",
+            "quy",
+            "rn",
+            "ro",
+            "ru",
+            "rw",
+            "sa",
+            "sat",
+            "sc",
+            "scn",
+            "sd",
+            "sg",
+            "shn",
+            "si",
+            "sk",
+            "sl",
+            "sm",
+            "sn",
+            "so",
+            "sq",
+            "sr-Cyrl",
+            "sr-Latn",
+            "ss",
+            "st",
+            "su",
+            "sv",
+            "sw",
+            "szl",
+            "ta",
+            "taq",
+            "te",
+            "tg",
+            "th",
+            "ti",
+            "tk",
+            "tl",
+            "tn",
+            "tpi",
+            "tr",
+            "ts",
+            "tt",
+            "tum",
+            "tw",
+            "tzm",
+            "ug",
+            "uk",
+            "umb",
+            "ur",
+            "uzn",
+            "vec",
+            "vi",
+            "war",
+            "wo",
+            "xh",
+            "ydd",
+            "yo",
+            "zh-cn",
+            "zh-tw",
+            "zsm",
+            "zu"
+        ],
+        "urls": [
+            "https://www.modernmt.com",
+            "https://www.modernmt.com/api#quality-estimation"
+        ],
+        "only_compatible_mt_api": null,
+        "only_compatible_tms": "matecat",
+        "customisation": true
+        
+    },
+    {   "name": "Quality Intelligence",
```

Comment:
I think it's more like

"name": "Unbabel Qi"
"id": "unbabel-qi",
"company": "unbabel",
"tagline": "Quality Intelligence AI"